website_rate_product/views/star_rate.xml

```diff
@@ -9,8 +9,8 @@
             </xpath>
         </template>
 
-        <template id="rating_product" inherit_id="website_sale.product" active="False" customize_show="True" name="Show Product Rating">
-            <xpath expr="//h1[@itemprop='name']" position="after">
+        <template id="rating_product" inherit_id="website_variants_extra.product_variants_description_3columns" active="False" customize_show="True" name="Show Product Rating">
```

Comment:
why did you remove this and added the other one?
@oscarolar ,Because, i replace all the right div and change the target to website_variants_extra, therefore this module depends of website_variants_extra